src/textual/css/stylesheet.py

```diff
@@ -108,25 +108,44 @@ def _check_rule(cls, rule: RuleSet, node: DOMNode) -> Iterable[Specificity3]:
                 yield selector_set.specificity
 
     def apply(self, node: DOMNode) -> None:
+        """
```

Comment:
Can you add a brief description in docstrings. Even if it seems a little redundant.